Source/VFECore/Genes/Harmony/PawnRenderer_GetBodyOverlayMeshSet.cs

```diff
@@ -56,10 +56,10 @@ public static class PawnRenderer_BaseHeadOffsetAt_Patch
 
         static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
         {
-            var headOffset = AccessTools.Field("BodyTypeDef:headOffset");
-            var pawn = AccessTools.Field("PawnRenderer:pawn");
-            var bodyScaleFactor = AccessTools.Method("PawnRenderer_BaseHeadOffsetAt_Patch:LifeStageFactorUpdated");
-            var codes = instructions.ToList();
+            FieldInfo headOffset = AccessTools.Field(typeof(BodyTypeDef), nameof(BodyTypeDef.headOffset));
+            FieldInfo pawn = AccessTools.Field(typeof(PawnRenderer), "pawn");
```

Comment:
```suggestion
            var headOffset = AccessTools.Field(typeof(BodyTypeDef), nameof(BodyTypeDef.headOffset));
            var pawn = AccessTools.Field(typeof(PawnRenderer), "pawn");
```
`var` is preferred, and beyond that, you should always keep your style consistent with the code you are modifying.

Reply:
Makes sense! I've reverted that change.
Personally, I am used to explicitly setting variable types because that helps my sanity, but your argument for keeping the style consistent makes sense.
Same for the line below.